recipes/asgi-csrf/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
can you get this other recipe out of here? seems irrelevant?

Reply:
yeah looks like I got confused managing my local branch